modules/cuml/src/utilities/array_utils.ts

```diff
@@ -0,0 +1,38 @@
+import {
+  DataFrame,
+  Int32,
+  Numeric,
+  Series,
+  TypeMap,
+} from '@rapidsai/cudf';
+import {DeviceBuffer} from '@rapidsai/rmm/';
+
+/**
+ * convert a dataframe to a single series to replicate conversion to a matrix in an organized
+ * for {x1,y1,x2,y2...} for df = {x:[x1,x2], y: [y1,y2]}
+ */
+export function dataframe_to_series(input: DataFrame): Series<Numeric> {
+  return input.interleaveColumns();
+}
```

Comment:
What's the concrete type this method returns? This means `Series<Bool8|Int8|Int16|Int32|Uint8|Uint16|Uint32|Int64|Uint64|Float32|Float64>`.